test/e2e/examples.go

```diff
@@ -48,24 +48,16 @@ except:
 	print 'err'`
 
 var _ = Describe("Examples e2e", func() {
+	framework := Framework{BaseName: "examples"}
```

Comment:
Use the constructor, it registers before & after each for you. You can register more than one if needed.